src/main/java/com/adyen/terminal/security/NexoCrypto.java

```diff
@@ -41,12 +41,14 @@
 import java.security.InvalidKeyException;
 import java.security.MessageDigest;
 import java.security.NoSuchAlgorithmException;
-import java.util.Random;
+import java.security.SecureRandom;
 
 import static com.adyen.model.terminal.security.NexoDerivedKey.NEXO_IV_LENGTH;
 
 public class NexoCrypto {
 
+    private static final SecureRandom SECURE_RANDOM = new SecureRandom();
```

Comment:
Perhaps consider replacing ```SecureRandom()``` with ```SecureRandom.getInstanceStrong()```

Reply:
@jillingk Very thanks for the suggestion
From what I'm reading SecureRandom.getInstanceStrong() uses dev/random which is blocked in Linux systems. I prefer to use the default one which uses dev/urandom and under the hood the getInstanceStrong is no secure than the default one.

Reply:
Hi @ykalayy, that makes sense. I will get this PR approved. Thanks again for contributing!